.travis.yml

```diff
@@ -16,11 +14,6 @@ matrix:
     include:
         - php: '7.3'
           env: TARGET=cs_dry_run
-        - php: '7.0'
-          env: COMPOSER_FLAGS="--prefer-lowest" SYMFONY_DEPRECATIONS_HELPER=weak
-        # test 3.4 LTS
-        - php: '7.1'
-          env: SYMFONY_VERSION=3.4.*
```

Comment:
Symfony 3.4 should still be tested IMO. Until November 2021: https://symfony.com/releases/3.4

Reply:
Oops, yes